pinot-controller/src/main/java/org/apache/pinot/controller/api/access/AccessControl.java

```diff
@@ -26,6 +26,9 @@
 @InterfaceAudience.Public
 @InterfaceStability.Stable
 public interface AccessControl {
+  String WORKFLOW_NONE = "NONE";
+  String WORKFLOW_BASIC = "BASIC";
+  String WORKFLOW_OAUTH2 = "OAUTH2";
```

Comment:
I guess we should stick to the ones that are already available in the code - ie none and basic. Whenever we add new implementation, we can add them here.
Also enum is a better choice than string.

Reply:
Hardcoding this in an enum would eliminate any extensibility of `AuthWorkflowInfo` by third parties